src/bucket/BucketList.cpp

```diff
@@ -892,6 +891,22 @@ BucketList::scanForEviction(Application& app, AbstractLedgerTxn& ltx,
         auto initialLevel = evictionIter.bucketListLevel;
         auto initialIsCurr = evictionIter.isCurrBucket;
         auto b = getBucketFromIter(evictionIter);
+
+        // Check to see if we can finish scanning the bucket before it receives
+        // an update
+        auto period = bucketUpdatePeriod(evictionIter.bucketListLevel,
```

Comment:
Is this is a protocol change?

Reply:
No, this only effects metrics and logging.